Bottom: 98f3cf3eb8418e00367fa8de9c2cc4605d6e45fd
Top:    b3d68627727f8c5cf7cc1b033b9fa5c00d9354a4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 02:15:48 +0100

Refresh of hid-gtk-ask-for-polygons-after

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 52d9990..8213bd0 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -270,8 +270,8 @@ ghid_use_mask (int use_it)
       break;
 
     case HID_MASK_BEFORE:
-      printf ("gtk doesn't support mask_before!\n");
-      abort ();
+      /* Ignore, this renderer doesn't use it */
+      return;
 
     case HID_MASK_CLEAR:
       if (!gport->mask)
